modules/integration/tests-ui/src/test/java/org/wso2/carbon/appmanager/integration/ui/TestCases/LifeCycleTestCase.java

```diff
@@ -92,13 +95,23 @@ public void testApplicationSubscribe() throws Exception {
         JSONObject subscriptionJsonObjectUnPublished = new JSONObject(subscriptionResponseUnpublished.getData());
         assertFalse((Boolean) subscriptionJsonObjectUnPublished.get("error"),
                 "Error while updating tier permission");
-        assertFalse((Boolean) subscriptionJsonObjectUnPublished.get("status"),
-                "Application "+appProp.getAppName()+" is already subscribed");
+        assertTrue((Boolean) subscriptionJsonObjectUnPublished.get("status"),
+                "Application " + appProp.getAppName() + " is already subscribed");
     }
 
     @AfterClass(alwaysRun = true)
     public void destroy() throws Exception {
         super.cleanup();
+        String parentWindow = driver.getWindowHandle();
+        Set<String> handles =  driver.getWindowHandles();
```

Comment:
This code segment has been refactored and this code line is not in here any more.